specification/ai/data-plane/Face/stable/v1.2/examples/LivenessSessionOperations_CreateLivenessSession.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "Create Liveness Session",
+  "operationId": "LivenessSessionOperations_CreateLivenessSession",
+  "parameters": {
+    "apiVersion": "v1.2",
+    "body": {
+      "livenessOperationMode": "Passive",
+      "deviceCorrelationIdSetInClient": true,
```

Comment:
This should be false if the device-correlation-id is going to be set when creating liveness session

Reply:
Fixed in all version.